content/partials/shared/_channel_enumeration.textile

```diff
@@ -25,4 +25,4 @@ Client libraries do not provide a dedicated API to enumerate channels, but make
 * Since the state of the cluster may change between successive calls, a pagination sequence may become invalid, in which case the request will respond with an error with code @40011@. In this case, to get a complete result, it is necessary to start the enumeration again from the beginning. Other API options to deal with this possibility maybe provided in later versions of this API. Enumerations that are satisfiable in the first response page do not have this issue.
 * The API does not guarantee that the limit will be achieved even if that would be possible. For example, if you specify a limit of 100, the API may return only 37 results together with a @next@ link to get the next page, even if you have more than 37 channels. In the extreme case, the API may return 0 results with a next link. In particular this may be the case if you have a large number of active channels but are specifying a @prefix@ that excludes a significant proportion of them.
 * The API does not guarantee that there will be no duplicated results between different pages, especially if a channel is alive in multiple regions. (It does not _currently_ do so, but it may begin to do so with no warning or deprecation period, so your implementation should be able to cope with duplication)
-* If you use @by=value@ (which until recent protocol versions was the default), just enumerating channels can briefly keep them alive, meaning if you do very regular enumeration you can get a situation where channels never close.
+* If you use @by=value@ in previous versions, just enumerating channels can briefly keep them alive, meaning if you do very regular enumeration you can get a situation where channels never close.
```

Comment:
> If you use `by=value` in previous versions,

This changes the meaning, which is now wrong. by=value doesn't only have this effect in previous versions, it has this effect in every version. the only thing the version changes is that by=value is no longer the default.